osp_checks.py

```diff
@@ -97,8 +100,48 @@ def check_volume_count(system, warn=10, crit=15, **kwargs):
     sys.exit(3)
 
 
+def check_services_status(system, **kwargs):
+    """Check to see if service are in the desired state"""
+    logger = kwargs["logger"]
+    kwargs.pop("logger", None)
+    hosts = list(set([host.host_name for host in system.api.hosts.list()]))
+    hosts_agents = dict()
+    hosts_status = dict()
+
```

Comment:
I think it is most clear to define the service dict here:
`services = kwargs["services"]`
and then refer to that below.